packages/website/pages/view/[id].tsx

```diff
@@ -30,9 +30,11 @@ const GET_OWNER_NFT = gql`
         media
         media_animation
       }
-      bids {
+      bids ( where: { accepted: false } ) {
```

Comment:
why not accepted bids?

Reply:
In the subgraph, we are saving bids for a token under the `token_id`. And right now, we are not removing a bid once it is accepted by the user. Instead of that, we are changing `accepted` attribute in the bids as `true`. So those accepted bids are gonna be there forever with the token. But we don't need to show those accepted bids to the current owner. Because we are only showing bids that can be accepted by the current owner. Hope it helps :)

Reply:
ah i see, i'll have to look better into it. seems like something is off.

Reply:
Sure. I think we can improve the subgraph structure for a better one too.